web-services-app/src/main/java/edu/unc/lib/boxc/web/services/rest/modify/StreamingPropertiesController.java

```diff
@@ -0,0 +1,94 @@
+package edu.unc.lib.boxc.web.services.rest.modify;
+
+import com.apicatalog.jsonld.StringUtils;
+import edu.unc.lib.boxc.auth.api.Permission;
+import edu.unc.lib.boxc.auth.api.models.AccessGroupSet;
+import edu.unc.lib.boxc.auth.api.models.AgentPrincipals;
+import edu.unc.lib.boxc.auth.api.services.AccessControlService;
+import edu.unc.lib.boxc.model.fcrepo.ids.PIDs;
+import edu.unc.lib.boxc.operations.jms.streaming.StreamingPropertiesRequest;
+import edu.unc.lib.boxc.operations.jms.streaming.StreamingPropertiesRequestSender;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import static edu.unc.lib.boxc.auth.fcrepo.services.GroupsThreadStore.getAgentPrincipals;
+import static edu.unc.lib.boxc.operations.jms.streaming.StreamingPropertiesRequest.ADD;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+/**
+ * Controller for setting streaming properties on a FileObject
+ */
+@Controller
+public class StreamingPropertiesController {
+    private static final Logger log = LoggerFactory.getLogger(StreamingPropertiesController.class);
+    private static final String ACTION = "action";
+    @Autowired
+    private AccessControlService accessControlService;
+    @Autowired
+    private StreamingPropertiesRequestSender streamingPropertiesRequestSender;
+
+    @PutMapping(value = "/edit/streamingProperties", produces = APPLICATION_JSON_VALUE)
+    @ResponseBody
+    public ResponseEntity<Object> updateStreamingProperties(@RequestParam Map<String,String> allParams) {
```

Comment:
Totally optional, but since we already have a handy request object we might actually be able to do 
```
public ResponseEntity<Object> updateStreamingProperties(StreamingPropertiesRequest streamingRequest) {
```
and have spring build the object for us (we'd still need to add the agent to it manually). Although I guess there is one mismatch with the FilePidString field being named "file" as a param currently. Could make them both the same, or rename it `id` everywhere, or just not worry about it and keep it the way it is.